content/millennium/dstu2/devices/device.md

```diff
@@ -7,10 +7,31 @@ title: Device | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Device Resource implementation is currently limited to devices implanted in a patient such as a pacemaker or insulin pump. As such, this resource currently exposes the known implant history for a patient at this site.
```

Comment:
Notice on the documentation for other resources we are inconsistent on whether or not "resource" should be capitalized. Seems like we tend toward upper case like we have here. Is there a preference?